src/replay/index.ts

```diff
@@ -1324,13 +1326,17 @@ export class Replayer {
               }
             }
           } else if (attributeName === 'style') {
-            for (var s in value) {
-              if (value[s] === false) {
-                ((target as Node) as Element).style.removeProperty(s);
-              } else if (Array.isArray(value[s])) {
-                ((target as Node) as Element).style.setProperty(s, value[s][0], value[s][1]);
+            let sv = (value as styleAttributeValue);
+            const htarget = ((target as Node) as HTMLElement);
+            for (var s in sv) {
+              if (sv[s] === false) {
+                htarget.style.removeProperty(s);
+              } else if (sv[s] instanceof Array) {
+                const svp = (sv[s] as styleValueWithPriority);
+                htarget.style.setProperty(s, svp[0], svp[1]);
```

Comment:
Tweaked naming to make it slightly clearer what is going on
```suggestion
            let styleValues = (value as styleAttributeValue);
            const targetEl = ((target as Node) as HTMLElement);
            for (var s in styleValues) {
              if (styleValues[s] === false) {
                targetEl.style.removeProperty(s);
              } else if (styleValues[s] instanceof Array) {
                const svp = (styleValues[s] as styleValueWithPriority);
                targetEl.style.setProperty(s, svp[0], svp[1]);
```